third_party/move/move-compiler/tests/move_check/parser/phantom_param_invalid_keyword.exp

```diff
@@ -2,7 +2,7 @@ error[E01002]: unexpected token
   ┌─ tests/move_check/parser/phantom_param_invalid_keyword.move:2:25
   │
 2 │     struct S<T1, phatom T2> {
-  │             -           ^ Expected '>'
+  │             -           ^ Expected either ',' or '>'
```

Comment:
This one is a bit weird, because it's just that phatom is not a keyword

Reply:
But it's not really terrible, and this is a small improvement in almost all cases, so maybe just approve?